### DISCUSSION TITLE
Send webhook after bookmarking the article

### DISCUSSION THREAD

**Discussion by marekbrze** [dailydotdev/daily#1328] [Feature Requests]
It would be great to be able to send article info using webhook after bookmarking the article. It would open a lot of possibilities like adding it to Todoist as a to-do

**Comment by nimrodkra** [comment]
@marekbrze we have a similar feature that uses rss instead of webhooks. You enable it you will need to enable bookmark sharing (public mode). Go to Bookmarks -> Click the sharing icon -> Switch public mode On.

![IMG_1014](https://github.com/user-attachments/assets/eab5ed10-339b-4799-bbf5-9e84a0ffce98)

**Reply by marekbrze** [reply]
Perfect! Thank you for the info. It's enough for my needs 🙌❤️
